lua/ttt2/libraries/entspawnscript.lua

```diff
@@ -353,7 +354,12 @@ if SERVER then
 	-- @internal
 	-- @realm server
 	function entspawnscript.ReadFile(dir)
-		return utilJSONToTable(fileRead(dir .. gameGetMap() .. ".json", "DATA"))
+		local fullDir = dir .. gameGetMap() .. ".json"
+		if fileExists(fullDir, "DATA") then
+			return utilJSONToTable( fileRead(fullDir, "GAME") )
```

Comment:
This should be `DATA` instead of `GAME`, right?

```suggestion
			return utilJSONToTable( fileRead(fullDir, "DATA") )
```